src/carousel.scss

```diff
@@ -0,0 +1,155 @@
+@import "./new-settings";
+@import "./mixins";
+
+$block: #{$fd-namespace}-carousel;
+$button: #{$fd-namespace}-button;
+
+.#{$block} {
+  $fd-carousel-page-indicator-border: 0.0625rem solid var(--sapPageFooter_BorderColor) !default;
+  $fd-carousel-button-size: 2.125rem !default;
+  $fd-carousel-button-content-offset: 0.5rem !default;
+  $fd-carousel-dot-size: 0.25rem !default;
+  $fd-carousel-dot-active-size: 0.5rem !default;
+
+  @include fd-reset();
+  @include fd-flex(column);
+
+  width: 100%;
+  min-width: 15.5rem;
+  max-width: 100%;
+  height: 100%;
+
+  &__content {
+    @include fd-reset();
+
+    flex: 1 1 auto;
+    width: 100%;
+    background: var(--sapBackgroundColor);
+    position: relative;
+
+    .#{$block}__button.#{$button} {
+      box-shadow: var(--sapContent_Shadow1);
+      position: absolute;
+      top: calc(50% - 1.0625rem);
+      margin: 0;
+
+      &.sap-icon--slim-arrow-left {
```

Comment:
We should not rely on icon class names. This will prevent users from changing the navigation icons.

Reply:
<img width="495" alt="Screen Shot 2020-08-26 at 9 38 52 AM" src="https://user-images.githubusercontent.com/39598672/91310521-faa1ec00-e77f-11ea-8c24-81c81a6c184b.png">

Fiori 3 specifies the icons that should be used in the navigation buttons so if someone wants to change them it will be against the guidelines. I am using the icons names to specify the horizontal offset of the buttons when used in content. We can't rely on first-child or last-child because they are using position: absolute. If we are not using the icon names we have to add additional classnames like `fd-carousel__button--left` and `fd-carousel__button--right`. Also these classnames will be used only when the buttons are inside the content, not when they are inside the Page indicator. So adding classes will mean that the buttons will either have different markup based on where they are used or the classnames will be redundant and do nothing when they are used in the Page Indicator. All these problems are solved with using icon names. Let me know what you think about it @salarenko @droshev @stefanoScalzo @katekozlowska @mikerodonnell89

Reply:
Yeah, but we can just always apply `fd-carousel__button--left` and `fd-carousel__button--right` and "conditionally" apply styles depending whether the buttons are placed inside `fd-carousel__content` or not, exactly at it is happening now. But I'm also curious what the others think.